apps/employee-monitoring/src/pages/settings/officer-of-the-day/index.tsx

```diff
@@ -50,6 +45,31 @@ const Index = () => {
     else return dayjs(date).format('MMMM DD, YYYY');
   };
 
+  // fetch data for list of user
+  const {
+    data: officersOfTheDay,
+    error: officersOfTheDayError,
+    isLoading: officersOfTheDayLoading,
+  } = useSWR('/officer-of-the-day', fetcherEMS, {
+    shouldRetryOnError: false,
+    revalidateOnFocus: false,
```

Comment:
Also removed this dependency. This table needs to be viewed always as an updated version. Also, add the mutate option for reloading upon successful add/delete of Officer.